openapi-generator/src/main/resources/templates/java-micronaut/common/model/pojo.mustache

```diff
@@ -382,9 +387,9 @@ public class {{classname}} {{#parent}}extends {{{parent}}} {{/parent}}{{#vendorE
     @Override
     public String toString() {
         return "{{classname}}("
-        {{#allVars}}
+        {{#vendorExtensions.allVars}}
```

Comment:
I think we should fix the `allVars` property instead of creating a new property in vendorExtensions.

Reply:
fixed